### PR TITLE
Turn on ALL the variants in Travis

### PR DESCRIPTION
**Pull request by tianon**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,14 +5,14 @@ dist: trusty
 
 env:
   - VERSION=3.4 VARIANT=
-#  - VERSION=3.4 VARIANT=slim
-#  - VERSION=3.4 VARIANT=wheezy
+  - VERSION=3.4 VARIANT=slim
+  - VERSION=3.4 VARIANT=wheezy
   - VERSION=3.3 VARIANT=
-#  - VERSION=3.3 VARIANT=slim
-#  - VERSION=3.3 VARIANT=wheey
```
Comment (Starefossen) [inline .travis.yml]: Wheey! :smile:

Comment (tianon) [inline .travis.yml]: Yeah...  :innocent:

**Comment by tianon** [comment]
:green_apple:

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**